hail/python/hailtop/utils/utils.py

```diff
@@ -66,6 +66,15 @@ def is_transient_error(e):
     # Connected call failed caused by:
     # OSError: [Errno 113] Connect call failed ('<ip>', 80)
     # 113 is EHOSTUNREACH: No route to host
+    #
+    # Fatal read error on socket transport
+    # protocol: <asyncio.sslproto.SSLProtocol object at 0x12b47d320>
+    # transport: <_SelectorSocketTransport fd=13 read=polling write=<idle, bufsize=0>>
+    # Traceback (most recent call last):
+    #   File "/anaconda3/lib/python3.7/asyncio/selector_events.py", line 812, in _read_ready__data_received
+    #     data = self._sock.recv(self.max_size)
+    # TimeoutError: [Errno 60] Operation timed out
```

Comment:
I don't think this is an OSError, I think this is a (Python builtin) TimeoutError:

```
$ python3
>>> TimeoutError
<class 'TimeoutError'>
```

However, it is the case that ClientOSError is just a subclass of OSError and ClientConnectionError, so I think could reasonably switch that case to just OSError.

Reply:
TimeoutError inherits from OSError.

Reply:
So maybe we just have OSError then.